pkg/controller/container-runtime-config/helpers_test.go

```diff
@@ -502,6 +502,44 @@ func TestUpdatePolicyJSON(t *testing.T) {
 			allowed:       []string{"allow.io"},
 			errorExpected: true,
 		},
+		{
+			name:    "allowed registries: atomic does not allow deep namespaces",
+			allowed: []string{"allow.io", "allowed-example.com/namespace1/namespace2/namespace3"},
```

Comment:
In both new test cases it would be useful to test that the two-namespace scopes are allowed.